app/controllers/concerns/saml_idp_logout_concern.rb

```diff
@@ -67,13 +67,17 @@ def prepare_saml_logout_response
     @saml_response = OneLogin::RubySaml::Logoutresponse.new(params[:SAMLResponse])
   end
 
+  def slo_session
```

Comment:
`slo` is single log-out, the counterpart in SAML to single sign on (sso).

e.g. http://help.sap.com/saphelp_nw73/helpdata/en/3f/8434a5048548eea5a970df57dadefb/content.htm